source/core/ut_metadata.pkb

```diff
@@ -134,12 +104,24 @@ create or replace package body ut_metadata as
   function get_package_spec_source(a_owner varchar2, a_object_name varchar2) return clob is
     l_txt_tab ut_varchar2_list;
     l_cur     sys_refcursor;
+    l_source  clob;
```

Comment:
yep fine.. good point. i'll re-add the back the blank lines.  the `get_package_spec_source` function is of course only returning the package spec. but the preprocessor can return the body.  as to which line numbers are returned we'd have to test.. but i would assume lines refer to the unprecessed source.. but since teh preprocessor just replaces with blanks that shoudl be fine still